src/panels/calendar/ha-recurrence-rule-editor.ts

```diff
@@ -129,7 +152,30 @@ export class RecurrenceRuleEditor extends LitElement {
   }
 
   renderMonthly() {
-    return this.renderInterval();
+    return html`
+      ${this.renderInterval()}
+      ${this._monthlyRepeatItems.length > 0
+        ? html`<ha-select
+            id="monthly"
+            label="Repeat Monthly"
```

Comment:
I just realized there are many labels in this file that are not yet using translations, so i'll update all of them in a followup pass.